src_c/surface.c

```diff
@@ -1609,6 +1648,26 @@ surf_set_alpha(pgSurfaceObject *self, PyObject *args)
     pgSurface_Prep(self);
     result =
         SDL_SetSurfaceRLE(surf, (flags & PGS_RLEACCEL) ? SDL_TRUE : SDL_FALSE);
+    /* HACK HACK HACK */
+    if ((surf->flags & SDL_RLEACCEL) && (!(flags & PGS_RLEACCEL)))
+    {
+        /* hack to strip SDL_RLEACCEL flag off surface immediately when
+           it is not requested */
+        sdlrect.x = 0;
+        sdlrect.y = 0;
+        sdlrect.h = 0;
+        sdlrect.w = 0;
+
+        surface = SDL_CreateRGBSurface(0, 1, 1, 32,
```

Comment:
I also didn't know whether this would be faster.

However it probably doesn't matter too much as the hack should right now only come into play in rare-ish circumstances and likely only in some kind of loading phase of a game - as you have to be first adding RLE acceleration to a surface and then removing it. This combination of circumstances likely only happens in solarwolf by accident, or perhaps because of behaviour that may just exist because of changes in SDL and/or pygame since solarwolf was created.